src/components/BasicHead.vue

```diff
@@ -244,6 +241,20 @@ export default {
   },
   
   methods: {
+    defaults () {
+      let features = ['print', 'bookmark', 'collect']
```

Comment:
concrete data needs to move to the model

Reply:
Removed - as it is not needed now